common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3078,4 +3079,14 @@ object CelebornConf extends Logging {
       .doc("Threads count for read buffer per mount point.")
       .intConf
       .createWithDefault(8)
+
+  val METRICS_EXTRA_LABELS: OptionalConfigEntry[Seq[String]] =
+    buildConf("celeborn.metrics.extra.labels")
```

Comment:
IMO it not valuable to introduce a new configuration namespace `celeborn.metrics.extra.`

```suggestion
    buildConf("celeborn.metrics.extraLabels")
```